src/test/java/org/springframework/data/ldap/repository/query/AnnotatedLdapRepositoryQueryIntegrationTests.java

```diff
@@ -141,27 +108,18 @@ EmbeddedLdapProperties embeddedLdapProperties() {
 	interface QueryRepository extends LdapRepository<SchemaEntry> {
 
 		@Query(value = "(cn=:fullName)")
-		List<SchemaEntry> namedParameters(@Param("fullName") String fullName, @Param("lastName") String lastName);
+		List<SchemaEntry> namedParameters(String fullName);
```

Comment:
Just FYI I've deliberately added two parameters so that we're sure that the proper one is picked.

Reply:
Indeed makes sense to have this test, see `AnnotatedLdapRepositoryQueryUnitTests.shouldEncodeBase`.

We like to keep integration test rather reduced to the minimum that is necessary. We tend to explore invariants in unit tests.